lingua_franca/time.py

```diff
@@ -17,16 +17,26 @@
 from dateutil.tz import gettz, tzlocal
 
 
+__default_tz = None
+
+
+def set_default_tz(tz):
```

Comment:
I'm not sure about that and i would like to tackle the whole config module in a different PR, me and @ChanceNCounter have been having some discussions on how config could look like

We may end up moving those, but I'd like to not do that for now since the config module is essentially a placeholder

Reply:
#128 definitely wants to be more than variables in a module, that's for sure.

Reply:
I agree with Gez on this one.  Why are we not just setting a config value?  I understand wanting to tackle it in a different PR.  Just curious the rationale for doing it this way.

Reply:
a proper config does not exist yet... PRs should do one thing only, let's not add stuff to things we know will be deprecated very soon unless absolutely necessary, dont want to have to support backwards compatibility, so let's keep this to a minimum before anyone starts depending on it

ideally we will have a config object later, right now we should do things with wrapper methods. supporting variables at the top level of a module is not good moving forward!

Reply:
Resolving this conversation as there seems to be a consensus that this will be addressed in a later revision.